integration/meta-typings/meta-typings-test.ts

```diff
@@ -0,0 +1,27 @@
+import { Child_Type, Simple, StateEnum } from './simple';
+
+const jan1 = new Date('1970-01-01T00:00:00.000Z');
+
+describe('simple', () => {
+  it('generates types correctly', () => {
```

Comment:
Can you add a test, maybe even an `toMatchInlineSnapshot` test, that asserts against the generated `metaPackageGoogleType`? To make sure all of the values match what you expect.